conf.yaml

```diff
@@ -241,6 +241,23 @@ contents:
               -
                 repo:   docs
                 path:   shared/settings.asciidoc
+          - title:      Elasticsearch Relevance Engine (ESRE)
+            prefix:     en/esre
+            index:      esre-docs/index.asciidoc
+            private:    1
+            current:    *stackcurrent
+            branches:   [ {main: master}, 8.9, 8.8 ]
+            live:       [ main, 8.9, 8.8 ]
```

Comment:
Oh.... just saw this — Should "main" be live?

We recently removed "main" from Stack live: https://github.com/elastic/docs/pull/2690

No worries if this is intended. Just a double-check.

Reply:
I think I'll change this. I can't use `stacklive` directly, because this content is 8.x only. I based the config on other books that aren't using `stacklive`, and they still contain `main`. However, I don't actually want to index main, so I'll push another commit to remove it. Thanks!